test/torchaudio_unittest/transducer_test.py

```diff
@@ -241,6 +242,14 @@ def compute_with_pytorch_transducer(data):
     return costs, gradients
 
 
+def skipIfNoRNNT(test_item):
```

Comment:
referring to "transducer" is more consistent with how we refer to it elsewhere
```suggestion
def skipIfNoTransducer(test_item):
```